DBSCE/Overview.md

```diff
@@ -0,0 +1,219 @@
+- [Kristian Monsen](), Google
+- [Phil Leblanc](), Google
+- [Sebastian](), Google
+- [Arnar Birgisson](), Google
+- [Pamela Dingle](), Microsoft
+- [Paul Garner](), Microsoft
+- [Erik Anderson](), Microsoft
+- [Will Bartlett](), Microsoft
+- [Kai Song](), Microsoft
+- [Amit Gusain](), Microsoft
+
+## Participate (TBD links)
+
+- [Issue tracker]()
+- [Discussion forum]
+
+## Overview
+
+Device Bound Session Credentials for Enterprise - DBSC(E), is an enhancement to the existing [DBSC](https://github.com/wicg/dbsc) proposal. It refines the key generation mechanism resulting in additional security for enterprise use cases. It aims to provide a mechanism for enterprise and advanced browser customers to be able to deploy true device binding for any browser session, hence protecting against session hijacking and credential theft.
+
+## Why DBSC(E)?
+
+While the original DBSC proposal is focused on providing a mechanism for browsers to bind session credentials to a device, it still remains vulnerable to malware that can run on a device during the initial web application signin/login. If a malware happens to be already running in the device, it can force the user to login, and provide its own credentials (asymmetric key pair) to the web application, there by gaining the ability to steal the session. Any upcoming sessions after this, even with DBSC, will not be reliable.
+
+DBSC(E) aims to mitigate this risk by introducing the concept of credential generation (asymmetric device-bound key) during the device registration and binds all the future sessions to the device. It guaratees any given session to be bound to the device, if the device registration is performed when there is no malware on the device (a state referred to as "clean room"). With DBSC(E), the malware will not be able to compromise a device even during signin/login. Device registration is also expected to be a once-in-a-lifetime operation, and hence the user will not be required to perform this operation again, reducing opportunities for malware to extract session credentials the device.
+
+## How does it integrate with DBSC?
+
+DBSC(E) is not intended to be a separate proposal from DBSC, it is rather building on existing DBSC, and adds the binding specific details to the protocol. It is expected that the DBSC(E) proposal will be integrated into the DBSC proposal in the specification. In the high-level design, we have folded the DBSC proposal into the end to end flow. Please read the [DBSC proposal](https://githuub.com/wicg/dbsc) before you proceed.
+
+Before we get into the specifics, we will introduce the terminology and design specifics for the key generation and validation below.
+
+## Terminology
+
+### Browser
+
+In this document, "Browser" refers to a functionality in a web browser that is responsible for the DBSC protocol. This functionality will be implemented by Edge, Chrome (or their common engine), and other browsers that choose to implement DBSC/DBSC(E).
```

Comment:
Nit:  "refers to a functionality" might read better as "refers to the functionality"

Reply:
Done